### ISSUE TITLE
PDFTemplates blank on edit

### ISSUE THREAD

**Issue by sergtech** [bitnami/bitnami-docker-suitecrm#63]
The PDFTemplates module has an issue with editing existing tamplates on brand new install using bitnami/suitecrm:7.9.4-r0.

Steps required:
1 Create new container
2 Create new PDF Template for Invoice/Quotes. Load the sample data for example
3 Try and edit the new template
4 The window is blank.

I could not find any suite_error logs (suitecrm loglevel set to error), or apache errors relating to this. The browser console is also clean. Data just vanishes.

I will investigate which version brings the error and post the simplest docker_compose.yml that replicates the issue shortly.
==============My previous docker-compose.yml file does it see below
version: '2'
services:
  mariadb_temptest2CRM:
    image: 'bitnami/mariadb:latest'
    volumes:
      - '/dev/shm/temptest2/mariadb_data:/bitnami/mariadb'
    environment:
      - MARIADB_ROOT_PASSWORD=somepassword
      - "TZ=Australia/Melbourne"
  suitecrm_temptescrm:
    image: 'bitnami/suitecrm:7.9.4-r0'
    ports:
      - '22223:443'
    volumes:
      - '/dev/shm/temptest2/suitecrm_data:/bitnami/suitecrm'
      - '/dev/shm/temptest2/apache_data:/bitnami/apache'
      - '/dev/shm/temptest2/php_data:/bitnami/php'
    environment:
      - SUITECRM_USERNAME=user
      - SUITECRM_PASSWORD=password
      - SUITECRM_EMAIL=someone@test.net
      - SUITECRM_LASTNAME=TempTestCRM
      - SUITECRM_HOST=10.0.0.127:22223
      - MARIADB_HOST=mariadb_temptest2CRM
      - MARIADB_PASSWORD=somepassword
      - "TZ=Australia/Melbourne"

    depends_on:
      - mariadb_temptest2CRM

**Comment by sergtech** [comment]
Interesting add-on note.
1. create instance using bitnami/suitecrm:7.8.3-r5
2. update .yml file to use bitnami/suitecrm:7.9.1
3. Template creation and edit works as expected
!! The User Profile does not have the "new" options of mail editor (e.g. Mozaik, TinyMCE, Direct HTML). I guess the includes for this may create the issue.

**Comment by javsalgar** [comment]
I was able to reproduce the issue. I saw that the issue did not appear in neither 7.9.1 nor 7.9.2 (it started with 7.9.3). I see that it only happens with the body but not with the header and footer (if you take a look closely you will see that the content is initially there and it disappears). I suspect that it must be an upstream regression, as we have not changed the way it is deployed/configured between versions 7.9.2-7.9.3. I suggest you to ask here: https://github.com/salesagility/SuiteCRM/issues
